vmdb/db/migrate/20150401090146_add_systemd_and_openstack_related_columns_to_system_services.rb

```diff
@@ -0,0 +1,9 @@
+class AddSystemdAndOpenstackRelatedColumnsToSystemServices < ActiveRecord::Migration
+  def change
+    add_column :system_services, :load, :string
+    add_column :system_services, :active, :string
+    add_column :system_services, :sub, :string
+    add_column :system_services, :openstack_host_service_id, :bigint
+    add_index  :system_services, :openstack_host_service_id
```

Comment:
Two notes:

1)  Petr has a [PR](https://github.com/ManageIQ/manageiq/pull/2554/) that takes care of the Openstack relationships.  So, let's remove those from there and work with him to get this PR and his PR working together.

2)  Maybe rename the other columns to prefix with `:systemd_`?  So, `:system_load`, `:systemd_active`, `:systemd_sub`.